neo/SmartContract/InteropService.cs

```diff
@@ -88,19 +88,21 @@ internal static bool Invoke(ApplicationEngine engine, uint method)
                 return false;
             if (!descriptor.AllowedTriggers.HasFlag(engine.Trigger))
                 return false;
+            if (descriptor.RequireWriteAccess && engine.CurrentContext.GetState<ExecutionContextState>().ReadOnly)
```

Comment:
No need to check this @shargon... if something really needs write access, you can invoke it, and let it fail naturally when it tries to access something that requires writing.
Only thing we need to ensure is: if current context is ReadOnly, then next invocations context will also be ReadOnly, but this is simple, as we can do this when loading new context into stack.

Reply:
I want to ban more syscalls like destroy and create, i think thats is better with this flag